src/fides/api/alembic/migrations/versions/1e750936a141_migrate_to_fideslang_2_0_uses_and_.py

```diff
@@ -0,0 +1,127 @@
+"""migrate to fideslang 2.0 uses and categories
+
+Revision ID: 1e750936a141
+Revises: fd52d5f08c17
+Create Date: 2023-08-17 08:51:47.226612
+
+"""
+import json
+from typing import Dict, List, Optional
+
+from alembic import op
+from loguru import logger
+from sqlalchemy import text
+from sqlalchemy.engine import Connection, ResultProxy
+from sqlalchemy.sql.elements import TextClause
+
+# revision identifiers, used by Alembic.
+revision = "1e750936a141"
+down_revision = "708a780b01ba"
+branch_labels = None
+depends_on = None
+
+###############
+## Data Uses ##
+###############
+# The `key` is the old value, the `value` is the new value
+data_use_upgrades: Dict[str, str] = {
+    "improve.system": "functional.service.improve",  # Verified in 2.0
+    "improve": "functional",  # Posted a question in the Confluence doc to verify this
+    "essential.service.operations.support.optimization": "essential.service.operations.improve",  # Verified in 2.0
+}
+data_use_downgrades: Dict[str, str] = {
+    value: key for key, value in data_use_upgrades.items()
+}
+
+#####################
+## Data Categories ##
+#####################
+# The `key` is the old value, the `value` is the new value
+data_category_upgrades: Dict[str, str] = {
+    "user.credentials": "user.authorization.credentials",  # Verified in 2.0
+    "user.observed": "user.behavior",  # Verified in 2.0
+    "user.browsing_history": "user.behavior.browsing_history",  # Verified in 2.0
+    "user.media_consumption": "user.behavior.media_consumption",  # Verified in 2.0
+    "user.search_history": "user.behavior.search_history",  # Verified in 2.0
+    "user.organization": "user.contact.organization",  # Verified in 2.0
+    "user.non_specific_age": "user.demographic.age_range",  # Verified in 2.0
+    "user.date_of_birth": "user.demographic.date_of_birth",  # Verified in 2.0
+    "user.gender": "user.demographic.gender",  # Verified in 2.0
+    "user.political_opinion": "user.demographic.political_opinion",  # Verified in 2.0
+    "user.profiling": "user.demographic.profile",  # Verified in 2.0
+    "user.race": "user.demographic.race_ethnicity",  # Verified in 2.0
+    "user.religious_belief": "user.demographic.religious_belief",  # Verified in 2.0
+    "user.sexual_orientation": "user.demographic.sexual_orientation",  # Verified in 2.0
+    "user.financial.account_number": "user.financial.bank_account",  # Verified in 2.0
+    "user.genetic": "user.health_and_medical.genetic",  # Verified in 2.0
+}
+data_category_downgrades: Dict[str, str] = {
+    value: key for key, value in data_use_upgrades.items()
+}
```

Comment:
I understand there's an outstanding TODO to handle data categories, but that brings up a similar question for data uses as well - do we need to migrate the existing `ctl_data_uses` and `ctl_data_categories` tables as well? Just trying to understand what's needed.  It's tricky when customers have also started building off of this taxonomy too.

Reply:
okay! Adam pointed me towards `load_default_taxonomy` which we already run on startup! I forgot about that. Brought fides down brought it back, verified the new ones were loaded, the old ones were left untouched. Nice to see that new `version_added` with `2.0.0` populated too, that was a good idea.

Reply:
yep, those tables should be good! That's what we have the most control over so we made it easier on ourselves